libp2p/transports/tcptransport.nim

```diff
@@ -62,78 +61,72 @@ proc setupTcpTransportTracker(): TcpTransportTracker =
 proc connHandler*(t: TcpTransport,
                   client: StreamTransport,
                   initiator: bool): Connection =
-  trace "handling connection", address = $client.remoteAddress
-  let conn: Connection = Connection(ChronosStream.init(client))
+  debug "Handling tcp connection", address = $client.remoteAddress,
+                                   initiator = initiator,
+                                   clients = t.clients[initiator].len
+
+  let stream = ChronosStream.init(client,
+                                  dir = if initiator: Direction.Out
```

Comment:
`Direction` would be better off without `None` I think - `Option[Direction]` can be used in place of `None`, where relevant